test/functional/outputs/forward_to_kafka_test.log

```diff
@@ -0,0 +1,175 @@
+package outputs
```

Comment:
should this be named "*.go"

Reply:
Looks like this file is still in progress... multiple errors etc .

Reply:
Also, without this file it is impossible to "execute" any tests so can't really test the functionality of this PR

Reply:
@pmoogi-redhat  please also move to `test/functional/outputs/kafka/....`